modules/home-manager/os76-imports/files/systemd-unit-browser.sh

```
@@ -0,0 +1,74 @@
+
+# Dependencies provided via Nix substitution
+GUM="@gum@"
+FZF="@fzf@"
+SYSTEMCTL="@systemctl@"
+JOURNALCTL="@journalctl@"
+SUDO="@sudo@"
+AWK="@awk@"
+
+# Function to list units and select one via fzf
+select_unit() {
+  "$SYSTEMCTL" list-units --all --no-legend | "$FZF" --height 40% --layout=reverse --border --header="Select a Systemd Unit" | "$AWK" '{print $1}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)